official/transformer/v2/transformer_main.py

```diff
@@ -177,23 +194,20 @@ def _create_callbacks(self, cur_log_dir, init_steps, params):
                                      params["hidden_size"],
                                      params["learning_rate_warmup_steps"])
     scheduler_callback = optimizer.LearningRateScheduler(sfunc, init_steps)
-
-    tb_logdir = os.path.join(cur_log_dir, "logs")
-    save_path = os.path.join(cur_log_dir,
-                             "weights-epoch-{epoch:02d}-loss-{loss:.4f}.hdf5")
-    csv_path = os.path.join(cur_log_dir, "result.csv")
-    return [
-        scheduler_callback,
-        tf.keras.callbacks.TensorBoard(tb_logdir),
-        tf.keras.callbacks.ModelCheckpoint(save_path, save_weights_only=True),
-        tf.keras.callbacks.CSVLogger(csv_path, append=True),
-    ]
+    callbacks = misc.get_callbacks()
+    callbacks.append(scheduler_callback)
+    ckpt_full_path = os.path.join(cur_log_dir, "cp-{epoch:04d}.ckpt")
+    callbacks.append(tf.keras.callbacks.ModelCheckpoint(ckpt_full_path,
+                                                        save_weights_only=True))
+    return callbacks
 
   def _load_weights_if_possible(self, model, init_weight_path=None):
     """Loads model weights when it is provided."""
     if init_weight_path:
       tf.compat.v1.logging.info("Load weights: {}".format(init_weight_path))
-      model.load_weights(init_weight_path, by_name=True)
```

Comment:
by_name is not needed anymore?

Reply:
Correct, that was with the other saving style.  I did multiple a 100K step run on 1 GPU and this works correctly.

Reply:
https://sponge.corp.google.com/target?id=48d603e1-252e-45c1-8385-52044f2c5be9&target=tensorflow_models%2Fperfzero%2Fmanual%2Fv100_1_gpu%2Fv100_1_gpu&searchFor=

Yup works perfectly.